vassal-app/src/main/java/VASSAL/tools/io/ZipArchive.java

```diff
@@ -47,6 +48,7 @@
 import java.util.zip.ZipOutputStream;
 
 import org.apache.commons.io.FileUtils;
+import VASSAL.i18n.Resources;
```

Comment:
Group this with the other `VASSAL.*` imports.

Reply:
Oh right - what you're seeing here is an IDE issue -- when we put in new things the IDE helpfully goes and puts the import right away so the code will compile, but it doesn't do a very good job of picking a spot, unfortunately. So if I forget to go manually check, this happens.

Reply:
One of the checkers has an import order rule, I believe, but I haven't found it yet.